app/javascript/shared/components/MarkdownEditor.res

```diff
@@ -243,9 +234,9 @@ type phraseModifer =
 
 let insertAndWrapper = phraseModifer =>
   switch phraseModifer {
-  | Bold => (`**${t("bold_insert")}**`, "**")
-  | Italic => (`*${t("italic_insert")}*`, "*")
-  | Strikethrough => (`~~${t("strikethrough_insert")}~~`, "~~")
```

Comment:
@indicozy - This file has changes unrelated to this PR. Could you please stash or move them to a new PR?